models/transformer.py

```diff
@@ -205,7 +209,10 @@ def call(self, inputs, training):
             output = self.decoder_dense[i](output)
         output = self.output_pred(self.output_dense(output))
 
-        return output
+        if self.output_attn:
```

Comment:
What if we store the attention scores in the `call()` method as an attribute of the `Transformer()` class, would it be possible to access them downstream for visualisation, etc? 

I think it would be a preferable solution, since we can then keep the output format intact (only predictions) -- so that there's no change needed in other modules because of the model interface change.